File: LeetCode/Contains_Duplicate.cpp
class Solution
{
public:
    bool containsDuplicate(vector<int> &arr)
    {
        sort(arr.begin(), arr.end());
        bool flag = false;
        for (int i = 0; i < arr.size() - 1; i++)
        {
            if (arr[i] == arr[i + 1])
                return true;
        }
        return flag;
    }
};

File: LeetCode/Contains_Duplicate.js
const containsDuplicate = (arr) => {
    arr.sort((a, b) => a - b)
    let flag = false;
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] == arr[i + 1]) {
            flag = true;
        }
    }
    return flag;
}

let arr = [1, 3, 2, 8, 0]
console.log(containsDuplicate(arr))